### ISSUE TITLE
erease supporters data

### ISSUE THREAD

**Issue by tttp** [fixthestatusquo/proca-server#50]
For privacy reasons and gdpr compliance, we need to delete the personal information about the supporters after 2 years (we need to double check and possibly adjust our own privacy policy)

However, some organisations might want to shorten the duration. As it's a destructive method, it is probably better if we provide two features:

- delete PII older than X months
- delete PII older than X months that are opt-out (rationale, there is no point to keep that information as long as the opt-in, they aren't going to be processed anyway

@marcinkoziej , we can delete the supporters data and keep the action data (eg "someone signed" on that date, but without knowing anymore who that someone was, right?

**Comment by marcinkoziej** [comment]
Yes PII data is decoupled. Also if data is shared between two orgs, one can delete earlier then other.

**Comment by tttp** [comment]
Cool, could you write a script eraseOld (param org + nb of month)?
